sapi/cli/tests/017.phpt

```diff
@@ -61,6 +61,8 @@ foreach ($codes as $key => $code) {
 }
 
 echo "\nDone\n";
+$dir = PHP_OS_FAMILY == 'Windows' ? getenv("USERPROFILE") : getenv("HOME");
```

Comment:
This test was merged in master. It should be adjusted to respect `PHP_HISTFILE`, when set.

Reply:
I've added some tests on the file with PHP_HISTFILE wdyt?

Reply:
ref https://github.com/php/php-src/pull/13967